llvm/lib/Support/APFloat.cpp

```diff
@@ -1007,6 +1023,10 @@ IEEEFloat &IEEEFloat::operator=(IEEEFloat &&rhs) {
 }
 
 bool IEEEFloat::isDenormal() const {
+  // No denormals in Float8E8M0FN
```

Comment:
A full sentence with a dot would be nice.

Reply:
Addressed in the latest revision.